ui/src/app/shared/service/service.ts

```diff
@@ -262,6 +291,92 @@ export class Service implements ErrorHandler {
     });
   }
 
+  public clickOtherPeriod() {
+    if (this.activePeriod === 'otherPeriod') {
+      this.setPeriod("today");
+    } else {
+      this.setPeriod("otherPeriod", this.YESTERDAY, this.TODAY);
+    }
+  }
+
+  public onDateRangeChanged(event: IMyDateRangeModel) {
+    console.log("ONDATERANGECHANGED")

```

Comment:
Wir hatten besprochen, dass in der service nur die allerwichtigsten Sachen stehen dürfen. Und wenn schon, sollten diese an dieser Stelle sehr gut kommentiert sein, sonst kennt sich hinterher niemand mehr aus. "console.logs" müssen vor dem Pull-Request alle raus.